src/internal/docker/cli.go

```diff
@@ -10,12 +10,11 @@ import (
 	"regexp"
 	"strings"
 
-	"github.com/mitchellh/go-homedir"
 	yey "github.com/silphid/yey/src/internal"
 	"github.com/silphid/yey/src/internal/logging"
 )
 
-func Start(ctx context.Context, yeyCtx yey.Context, containerName string) error {
+func Start(ctx context.Context, yeyCtx yey.Context, containerName, workDir string) error {
```

Comment:
This is an aesthetic point, but it feels like workDir should be an option.
To keep our docker package `sexy` I would have the signature be: 

`func Start(ctx context.Context, yeyCtx yey.Context, containerName string, opts ...StartOption)`

and the call site would be something like:

```go
docker.Start(ctx, yeyCtx, name, docker.Workdir(wd))
```